src/utils/promises/promises.js

```diff
@@ -1,3 +1,2 @@
-import promise from 'es6-promise';
-
-promise.polyfill();
+import 'core-js/es6/promise';
+import 'core-js/fn/array/find';
```

Comment:
could remove this if you use lodash method instead https://lodash.com/docs/4.17.4#find